modules/ascat/main.nf

```diff
@@ -165,26 +167,27 @@ process ASCAT {
 
     """
 
-
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    echo stub > ${prefix}.cnvs.txt
-    echo stub > ${prefix}.metrics.txt
-    echo stub > ${prefix}.purityploidy.txt
-    echo stub > ${prefix}.segments.txt
-    echo stub > Tumour.ASCATprofile.png
-    echo stub > Tumour.ASPCF.png
-    echo stub > Before_correction_Tumour.germline.png
-    echo stub > After_correction_GC_Tumour.germline.png
-    echo stub > Tumour.rawprofile.png
-    echo stub > Tumour.sunrise.png
-    echo stub > Before_correction_Tumour.tumour.png
-    echo stub > After_correction_GC_Tumour.tumour.png
-    echo stub > Tumour_alleleFrequencies_chr21.txt
-    echo stub > Tumour_alleleFrequencies_chr22.txt
-    echo stub > Normal_alleleFrequencies_chr21.txt
-    echo stub > Normal_alleleFrequencies_chr22.txt
+    echo stub > ${prefix}_after_correction_gc_rt_test_tumour.germline.png
+    echo stub > ${prefix}_after_correction_gc_rt_test_tumour.tumour.png
+    echo stub > ${prefix}_before_correction_test_tumour.germline.png
+    echo stub > ${prefix}_before_correction_test_tumour.tumour.png
+    echo stub > ${prefix}_cnvs.txt
+    echo stub > ${prefix}_metrics.txt
+    echo stub > ${prefix}_normal_alleleFrequencies_chr21.txt
+    echo stub > ${prefix}_normal_alleleFrequencies_chr22.txt
+    echo stub > ${prefix}_purityploidy.txt
+    echo stub > ${prefix}_segments.txt
+    echo stub > ${prefix}_tumour.ASPCF.png
+    echo stub > ${prefix}_tumour.sunrise.png
+    echo stub > ${prefix}_tumour_alleleFrequencies_chr21.txt
+    echo stub > ${prefix}_tumour_alleleFrequencies_chr22.txt
+    echo stub > ${prefix}_tumour_normalBAF.txt
+    echo stub > ${prefix}_tumour_normalLogR.txt
+    echo stub > ${prefix}_tumour_tumourBAF.txt
+    echo stub > ${prefix}_tumour_tumourLogR.txt
```

Comment:
```suggestion
    echo stub > ${prefix}.after_correction.gc_rt_test_tumour.germline.png
    echo stub > ${prefix}.after_correction.gc_rt_test_tumour.tumour.png
    echo stub > ${prefix}.before_correction.test_tumour.germline.png
    echo stub > ${prefix}.before_correction.test_tumour.tumour.png
    echo stub > ${prefix}.cnvs.txt
    echo stub > ${prefix}.metrics.txt
    echo stub > ${prefix}.normal.alleleFrequencies_chr21.txt
    echo stub > ${prefix}.normal.alleleFrequencies_chr22.txt
    echo stub > ${prefix}.purityploidy.txt
    echo stub > ${prefix}.segments.txt
    echo stub > ${prefix}.tumour.ASPCF.png
    echo stub > ${prefix}.tumour.sunrise.png
    echo stub > ${prefix}.tumour.alleleFrequencies_chr21.txt
    echo stub > ${prefix}.tumour.alleleFrequencies_chr22.txt
    echo stub > ${prefix}.tumour.normalBAF.txt
    echo stub > ${prefix}.tumour.normalLogR.txt
    echo stub > ${prefix}.tumour.tumourBAF.txt
    echo stub > ${prefix}.tumour.tumourLogR.txt
```

Reply:
if that works with how ascat names the files

Reply:
Almost